package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "root",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/@vue",
+      "**/@vue/**",
+      "**/vue-cli-plugin-vuetify",
+      "**/vue-cli-plugin-vuetify/**",
+      "**/vuetify-loader",
+      "**/vuetify-loader/**",
+      "**/vue-jest",
+      "**/vue-jest/**",
+      "**/vue-loading-overlay",
+      "**/vue-loading-overlay**",
+      "**/@mdi",
+      "**/@mdi/**",
+      "**/material-design-icons-iconfont",
+      "**/material-design-icons-iconfont/**",
+      "**/@sap-devx",
+      "**/@sap-devx/**"
+    ]
+  },
+  "scripts": {
+    "release:version": "lerna version",
+    "release:publish": "lerna publish from-git --yes",
+    "ci": "npm-run-all coverage:clean format:validate lint:validate legal:* ci:subpackages coverage:merge",
+    "ci:subpackages": "lerna run ci",
+    "compile": "lerna run clean && tsc --build",
+    "compile:watch": "lerna run clean && tsc --build --watch",
+    "format:fix": "prettier --write \"**/*.@(js|ts|json|md|vue)\" --ignore-path=.gitignore",
+    "format:validate": "prettier --check \"**/*.@(js|ts|json|md|vue)\" --ignore-path=.gitignore",
+    "lint:fix": "eslint . --ext=js,ts,vue --fix --max-warnings=0 --ignore-path=.gitignore",
+    "lint:validate": "eslint . --ext=js,ts,vue --max-warnings=0 --ignore-path=.gitignore",
+    "coverage:clean": "rimraf ./coverage",
+    "coverage:merge": "lcov-result-merger \"./coverage/lcov*.info\" \"./coverage/lcov.info\"",
+    "legal:delete": "lerna exec \"shx rm -rf .reuse LICENSES\" || true",
+    "legal:copy": "lerna exec \"shx cp -r ../../.reuse .reuse && shx cp -r ../../LICENSES LICENSES\""
+  },
+  "dependencies": {
+    "lodash": "4.17.21"
```

Comment:
do you have a dep to lodash here?
it is possible you no longer due if you have fewer scripts than the original ECMA mono repo template.

Reply:
Not sure I understand this comment.